src/pages/create-new-checkin-page/steps/review-previous-checkin-step.jsx

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  checkin: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    user: PropTypes.string.isRequired,
+    createdAt: PropTypes.number.isRequired,
+    tasks: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+        completed: PropTypes.bool.isRequired,
+      })
+    ).isRequired,
+    blockers: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+      })
+    ).isRequired,
+  }),
+  onComplete: PropTypes.func.isRequired,
+};
+
+const ReviewPreviousCheckinStep = ({ checkin, onComplete }) => {
+  const { tasks, blockers } = checkin;
+
+  return (
+    <div>
+      <section>
+        <p>Your last checkin said you would:</p>
```

Comment:
Do we need this text or different text, maybe, if there were no previous tasks?

Reply:
Added a block above to handle no previous checking